src/repairnator-pipeline/pom.xml

```diff
@@ -20,9 +20,18 @@
             <name>tdurieux.github.io maven-repository</name>
             <url>https://tdurieux.github.io/maven-repository/releases/</url>
         </repository>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
+        </repository>
     </repositories>
-    
+
     <dependencies>
+        <dependency>
```

Comment:
don't import spoon directly, it is obtained transitively

Reply:
I need to make to PR to spoon labs :) . Its spoon version (7.3.0)  is incompatible with Sonarqube Repair (8.2.0-beta-2)

Reply:
I ran it locally and it seems to be a great deal of work to update spoon core

Reply:
OK.It does not make sense to deploy sonarqube-repair to https://repo.jenkins-ci.org/public/.

We will deploy it on Maven Central directly, see https://github.com/kth-tcs/sonarqube-repair/pull/73

Please remove this repo